webpage/app/page.tsx

```
@@ -0,0 +1,123 @@
+    setVertices(Array.from(Array(adjacencyMatrix.length).keys()).map(i => (i + 1).toFixed(0)));
+
+  }, [adjacencyMatrix])
+
+  const [vertices, setVertices] = useState(Array.from(Array(adjacencyMatrix.length).keys()).map(i => (i + 1).toFixed(0)));
+  const [showInfo, setShowInfo] = useState(false);
+
+  return (
+    <main className="flex h-screen flex-col items-center font-sans">
+      <nav className="flex items-center justify-between w-screen bg-slate-100 p-5">
+        <a href="#" className="text-gray-900 font-medium flex flex-row gap-2 items-center"><Image alt="TUM" src={"tum_logo.svg"} width={60} height={1}></Image>MQT QUBOMaker: Pathfinder</a>
+        <a target="_blank" href="https://www.cda.cit.tum.de/research/quantum/" className="text-gray-900 font-medium hidden md:block"><Image alt="MQT" src={"mqt_dark.png"} width={120} height={1} unoptimized></Image></a>
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)